src/commands/flutter-symbols/upload.ts

```diff
@@ -0,0 +1,468 @@
+import fs from 'fs'
+
+import {Command} from 'clipanion'
+import yaml from 'js-yaml'
+import {
+  renderArgumentMissingError,
+  renderCommandInfo,
+  renderCommandSummary,
+  renderFailedUpload,
+  renderGeneralizedError,
+  renderGitWarning,
+  renderInvalidPubspecError,
+  renderInvalidSymbolsDir,
+  renderMissingAndroidMappingFile,
+  renderMissingDartSymbolsDir,
+  renderMissingPubspecError,
+  renderPubspecMissingVersionError,
+  renderRetriedUpload,
+  renderUpload,
+  renderVersionBuildNumberWarning,
+  UploadInfo,
+} from './renderer'
+
+import glob from 'glob'
+import asyncPool from 'tiny-async-pool'
+import {ApiKeyValidator, newApiKeyValidator} from '../../helpers/apikey'
+import {getRepositoryData, RepositoryData} from '../../helpers/git/format-git-sourcemaps-data'
+import {getMetricsLogger, MetricsLogger} from '../../helpers/metrics'
+import {MultipartValue, UploadStatus} from '../../helpers/upload'
+import {buildPath, DEFAULT_CONFIG_PATH, performSubCommand, resolveConfigFromFile} from '../../helpers/utils'
+import * as dsyms from '../dsyms/upload'
+import {newSimpleGit} from '../git-metadata/git'
+import {getArchInfoFromFilename, getFlutterRequestBuilder, uploadMultipartHelper} from './helpers'
+import {
+  DART_SYMBOL_FILE_NAME,
+  JVM_MAPPING_FILE_NAME,
+  MappingMetadata,
+  TYPE_DART_SYMBOLS,
+  TYPE_JVM_MAPPING,
+  VALUE_NAME_DART_MAPPING,
+  VALUE_NAME_JVM_MAPPING,
+} from './interfaces'
```

Comment:
Convention in this repo is to group imports by:
- first party imports (only `fs` in that case)
- third party imports (`clipanion`, `js-yaml`, `glob`, `tiny-async-pool`)
- project imports with pathes with the most `../` first

Unfortunately `tslint` does not support checking for that now, I've created a card to move `datadog-ci` to `eslint` which will also support this rule.

Reply:
👍 I'll move around imports.